common/src/test/scala/org/apache/celeborn/ConfigurationSuite.scala

```diff
@@ -139,8 +140,8 @@ class ConfigurationSuite extends AnyFunSuite {
   }
 
   def appendConfigurationTableHeader(output: ArrayBuffer[String]): Unit = {
-    output += "| Key | Default | Description | Since | Deprecated |"
-    output += "| --- | ------- | ----------- | ----- | ---------- |"
+    output += "| Key | Default | isDynamic | Description | Since | Deprecated |"
```

Comment:
Why does the document show whether the config is dynamic? IMO, all configs could be dynamic at default.